### DISCUSSION TITLE
[Tracking] Higher value loot to put into mimics

### DISCUSSION THREAD

**Discussion by Sollace** [Sollace/Unicopia#594] [Ideas]
I want to make it so mimic chests generate with higher value loot in them. So for example, when you open a chest for the first time and the game decides it's going to be a mimic, it would insert an additional (1) item in there that would make fighting the mimic worth it.

Y'all are welcome to suggest the items that should be included, so long as it's something that doesn't normally appear in generated chests or is more valuable than what you'd normally expect to see.

**Comment by Sollace** [comment]
One of the ideas is to have enchantments that are inversions/variants of the typical enchants.

i.e. Ice Aspect and Lava Walker

Some more options:
- Axe with Bleeding: Hitting enemies heals you
- Sword with Sucking II: Swinging without hitting anything pulls mobs towards you
- A shovel (or rusty frying pan) with percussion: Hitting a mob creates a loud "DONG" and a pressure wave that hurts all surrounding enemies for smaller amounts the further they are away from you.

**Comment by EternaDash** [comment]
Here's Two Other Inverted Enchants,

Rot [?]
-
Max Level: V
Inverse Of: Smite
Effect: Deals Great Damage To Living Entities, Like Pigs, Cows, Villagers, Pillagers, And The Likes. (not other ponies, that'd throw off pvp)

Light
-
Max Effect: I
Inverse Of: Heavy
Effect: Increases The Speed Of A Pony Wearing It.
